File: remidial - 2/Poll/Participant.java
class Participant{
	private String name;
	private int option;
	private String optedname;

	public Participant(String name,int option,String optedname){
		this.name = name;
		this.option = option;
		this.optedname = optedname;

	}
	 public String getName(){
	  return name;
	 }
	 
	 public void setName(String name){
	  this.name = name;
	 }
	  public int getOption(){
	   return option;
	  }
	  
	  public void setOption(int option){
	   this.option = option;
	  }
	   public String getOptedname(){
	    return optedname;
	   }
	   
	   public void setOptedname(String optedname){
	    this.optedname = optedname;
	   }
}

File: remidial - 2/my sol/Poll/Participant.py
class Participant(object):
	"""docstring for Participant"""
	def __init__(self,name,option,optedName):
		# super(Participant, self).__init__()
		self.name = name
		self.option = option
		self.optedName = optedName

	def getName(self):
		return self.name
	def setName(self,newname):
		self.name = newname

	def getOption(self):
		return self.option
	def setOption(self,newoption):
		self.option = newoption

	def getOptedname(self):
		return self.optedName
	def setOptedname(self,newoptedname):
		self.optedName = newoptedname
